### PR TITLE
SEMI baseline: grouping based on size 

### PR DESCRIPTION
**Pull request by KatGarmash**
implements one of the grouping criterion for extract opportunities: similar size (line 7, Fig. 7 from the paper)

### CODE REVIEW

**Comment by lyriccoder** [comment]
I would add License info. Should we do it? @acheshkov

**Comment by acheshkov** [comment]
@lyriccoder we have MIT license already

File: test/baselines/semi/test_grouping_size.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class SimilarSizeTest(TestCase):
+
+    def test_is_similar_size(self):
+        example_1_1 = (2, 32)
+        example_1_2 = (2, 34)
+        self.assertEqual(is_similar_size(example_1_1, example_1_2), True)
+
+        max_diff = 0.05
+        self.assertEqual(is_similar_size(example_1_1, example_1_2, max_size_difference=max_diff),
+                         False)
```
Comment (acheshkov) [inline test/baselines/semi/test_grouping_size.py:15]: Here are two different tests.

**Review by acheshkov — CHANGES_REQUESTED** [review]

File: veniq/baselines/semi/grouping_size.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from typing import Tuple
+
+
+def is_similar_size(range_1: Tuple[int, int],
+                    range_2: Tuple[int, int],
+                    max_size_difference: float = 0.2) -> bool:
+    """
```
Comment (acheshkov) [inline veniq/baselines/semi/grouping_size.py:7]:
```suggestion
def is_similar_size(
    range_1: Tuple[int, int],
    range_2: Tuple[int, int],
    max_size_difference: float = 0.2
) -> bool:
    """

```

File: test/baselines/semi/test_grouping_size.py (hunk 1 above)
Comment (acheshkov) [reply test/baselines/semi/test_grouping_size.py:15]: split on two tests

Comment (KatGarmash) [reply test/baselines/semi/test_grouping_size.py]: fixed

**Review by acheshkov — APPROVED** [review]

**Pull request merged**